keylime/cmd_exec.py

```diff
@@ -12,7 +12,7 @@
 
 
 def _execute(cmd, env=None, **kwargs):
-    proc = subprocess.Popen(cmd, env=env, stdout=subprocess.PIPE,
+    proc = subprocess.Popen(cmd, env=env, stdout=subprocess.PIPE,  #pylint: disable=consider-using-with
```

Comment:
Why does with not work here?

Reply:
Due to the test failures I have been rolling things back that I am not sure about. I will actually start bisecting through the changes to see where the tests are failing. I don't see the reason. So I'll keep the CI/CD busy for a while...